iOS/DuckDuckGo/DuckPlayer/DuckPlayerNavigationHandler.swift

```diff
@@ -81,7 +81,30 @@ final class DuckPlayerNavigationHandler: NSObject {
     
     /// Cancellable for observing DuckPlayer Navigation Request
     private var duckPlayerNavigationRequestCancellable: AnyCancellable?
+        
+    /// Cancellable for observing DuckPlayer dismissal
+    private var duckPlayerDismissalCancellable: AnyCancellable?
     
+    /// JavaScript for media playback control        
```

Comment:
These are the two local scripts we inject to control muting and pause Youtube's videos.   Eventually these should be moved to CSS if they decide to use in other platforms, but for now, we're just injecting them in Youtube watch pages.

Reply:
Were the scripts reviewed by a FE dev? If not, I believe they should just to make sure there isn't any gotchas.

Reply:
FE folk knows about them.  I'll loop them in the PR